src/components/shared/Mod.tsx

```diff
@@ -94,7 +94,7 @@ export default function Mod({
           <div style={{ fontSize: '.8rem', color: colors.text.secondary }}>
             <LockOutlined /> <Trans>until</Trans>{' '}
             {mod.lockedUntil
-              ? formatDate(mod.lockedUntil * 1000, 'MM-DD-yyyy')
```

Comment:
Note these format changes throughout this PR. This is to better internationalize the site - yyyy-mm-dd is the international standard: https://www.cl.cam.ac.uk/~mgk25/iso-time.html#:~:text=YYYY%2DMM%2DDD,month%20between%2001%20and%2031.

cc @peripheralist for awareness